app/views/books/show.html.erb

```diff
@@ -27,7 +27,6 @@
     <li><%= author.name %></li>
   <% end %>
 </ul>
-<img src="<%= @book.image_url %>">
 <h4>Reviews:</h4>
 <p>
   <% @book.reviews.each do |review| %>
```

Comment:
Accidentally created two images on the same show page working separately. Fixed by deleting one.